### PR TITLE
add note about pysss for pam eauth

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
Add note

### CODE REVIEW

File: salt/auth/pam.py [hunk 1]
```diff
@@ -23,6 +23,14 @@
 
     The Python interface to PAM does not support authenticating as ``root``.
 
+.. note:: Using Pam groups with SSSD groups on python2.
```
Comment (cachedout) [inline salt/auth/pam.py]: `PAM` not `Pam`

File: salt/auth/pam.py [hunk 2]
```diff
@@ -23,6 +23,14 @@
...
+
+    To use sssd with the pam eauth module and groups the `pysss` module is
```
Comment (cachedout) [inline salt/auth/pam.py]: `PAM`

**Review by cachedout — CHANGES_REQUESTED** [review]

**Comment by gtmanfred** [comment]
updated

**Comment by cachedout** [comment]
Thank you very much, @gtmanfred

**Pull request merged**